### ISSUE TITLE
Roll Templates not working no matter what I do!

### ISSUE THREAD

**Issue by Elblindside** [kakaroto/Beyond20#741]
I cannot get the Roll Template to work for 5E by Roll20 using Beyond20 no matter what I do, it comes up with the ugly purple bar!

Using firefox and have the latest beyond20 downloaded. I originally installed the github version before the latest update to avoid this and am wondering if that's affecting it now?

**Comment by Elblindside** [comment]
Hey, this still isn't working for me. I tried to install the dev version following the instructions as above and it's not coming up properly. Is there something I'm missing? I followed all the steps as mentioned in that comment, and several months ago this worked fine for me but now it isn't at all.

**Comment by Aeristoka** [comment]
> Hey, this still isn't working for me. I tried to install the dev version following the instructions as above and it's not coming up properly. Is there something I'm missing? I followed all the steps as mentioned in that comment, and several months ago this worked fine for me but now it isn't at all.

Did you disable the Release version?

**Comment by Elblindside** [comment]
Yeah, disabled the release verion. It now comes up with this in my toolbar and won't allow me to access the options and it also isn't coming up on Beyond or Roll20
![Screen Shot 2021-03-30 at 13 58 18](https://user-images.githubusercontent.com/81628384/112992377-06afbb00-9160-11eb-9720-f3507a0c5027.png)

**Comment by Elblindside** [comment]
I installed both Manifest.json and Manifest_ff.json just to be sure, tried them seperately and neither worked. Followed literally every step on that thread and still got problems. :/

**Comment by Elblindside** [comment]
![Screen Shot 2021-03-30 at 14 00 22](https://user-images.githubusercontent.com/81628384/112992842-78880480-9160-11eb-8b23-62ff17bffb62.png)

**Comment by Aeristoka** [comment]
I can't replicate the issue you're having... It worked for me first time...

**Comment by kakaroto** [comment]
> I installed both Manifest.json and Manifest_ff.json just to be sure, tried them seperately and neither worked. Followed literally every step on that thread and still got problems. :/

This tells me you downloaded from the source, which isn't compiled, so that might be why it failed. The firefox.zip file should only have a `manifest.json` for firefox, also note that firefox will always use `manifest.json` in that same directory even if you select a different file when doing a load.. it kind of ignores the file you select, instead looking at the folder where the file is.
Anyways, just wanted to make sure you were aware that the release was published a couple of days ago, so you should be fine to switch to the latest release instead.

**Comment by Elblindside** [comment]
Hey, was workign great for a few days thanks! But now has broken again, I assume this is Roll20 messing up again?

**Comment by amenokos** [comment]
When I went to play this evening (about 3 hours ago), 2.4.0 release was not working and the purple default boxes were showing up in Roll20's game log. I disabled it and installed the current dev, and that seems to be working for now.

**Comment by Aeristoka** [comment]
> When I went to play this evening (about 3 hours ago), 2.4.0 release was not working and the purple default boxes were showing up in Roll20's game log. I disabled it and installed the current dev, and that seems to be working for now.

Current dev does indeed have a fix for it.

If only Roll20 would stop fiddling with crap on their end...
